ThunderCatsTests/UITextFieldTests.m

```diff
@@ -0,0 +1,37 @@
+//
+//  UITextFieldTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-12.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UITextField+TCAdditions.h"
+
+@interface UITextFieldTests : XCTestCase
+
+@end
+
+@implementation UITextFieldTests
+
+- (void)testSetRightAndLeftPadding
+{
+    UITextField *textField = [[UITextField alloc] initWithFrame:CGRectMake(10, 10, 10, 10)];
+    [textField tc_setLeftAndRightPadding:15];
+    
+    XCTAssertEqual(textField.frame.size.height, 10);
```

Comment:
Because these are floating point numbers, we should probably be using `XCTAssertEqualWithAccuracy`.